opal/mca/event/external/configure.m4

```diff
@@ -115,11 +116,41 @@ AC_DEFUN([MCA_opal_event_external_CONFIG],[
                               [opal_event_external_support=yes],
                               [opal_event_external_support=no])
 
+           # Check to see if this check failed because it conflicted with LSF's libevent.so
+           # This can happen if LSF's library is in the LDFLAGS envar or default search
+           # path. The 'event_fini' function is only defined in LSF's libevent.so and not
+           # in Libevent's libevent.so
+           AS_IF([test "$opal_event_external_support" = "no"],
+                 [AC_CHECK_LIB([event], [event_fini],
+                               [AC_MSG_WARN([===================================================================])
+                                AC_MSG_WARN([Possible conflicting libevent.so libraries detected on the system.])
+                                AC_MSG_WARN([])
+                                AC_MSG_WARN([LSF provides a libevent.so that is not from Libevent in its])
+                                AC_MSG_WARN([library path. It is possible that you have installed Libevent])
+                                AC_MSG_WARN([on the system, but the linker is picking up the wrong version.])
+                                AC_MSG_WARN([])
+                                AC_MSG_WARN([Configure will continue and attempt to use the 'internal' libevent])
+                                AC_MSG_WARN([instead of the 'external' libevent.])
+                                AC_MSG_WARN([])
+                                AC_MSG_WARN([If your intention was to use the 'external' libevent then you need])
+                                AC_MSG_WARN([to address this linker path ordering issue. One way to do so is])
+                                AC_MSG_WARN([to make sure the libevent system library path occurs before the])
+                                AC_MSG_WARN([LSF library path.])
+                                AC_MSG_WARN([===================================================================])
+                                opal_event_external_support=no
+                               ],
+                               [])
+                 ])
+
+           AS_IF([test "$opal_event_external_support" = "yes"],
+                 [LDFLAGS="$opal_event_external_LDFLAGS $LDFLAGS"
+                  CPPFLAGS="$opal_event_external_CPPFLAGS $CPPFLAGS"],
+                 [])
```

Comment:
You don't need to put the empty `[]` here.

Reply:
Nit: you still have the empty `[]` there.  It's not a deal-breaker, but I think you meant to take it out and missed it.  😇

Reply:
It was removed. you need to refresh the browser.